docs/docs/reference/custom-components/05-aica-package-toml.md

```diff
@@ -167,7 +164,7 @@ Because `package-builder` will use your dependencies in the metadata of the comp
 
 ```toml title="aica-package.toml"
 [build.dependencies]
-"@aica/foss/control-libraries" = {version = "v7.5.0", constraints = ">= 7.5.0"}
+"@aica/foss/control-libraries" = {version = "v9.0.1", constraints = ">= 9.0.1"}
```

Comment:
@LouisBrunner is it correct that the version is with the `v` prefix but the constraint not?